public/app/features/dashboard/components/DashboardSettings/GeneralSettings.tsx

```diff
@@ -115,6 +117,12 @@ export function GeneralSettingsUnconnected({
             />
           </Field>
 
+          {config.featureToggles.nestedFolderPicker && (
+            <Field label="Nested Folder">
+              <NestedFolderPicker />
+            </Field>
```

Comment:
This naming is very strange. You don't select a nested folder you select a folder. This should not be an extra option